### PR TITLE
disabled failing tests in vs code

### LINKED ISSUE #7573: VSCode test failure
These tests failed on build 41090.

image

### PR DESCRIPTION
**Pull request by ellbosch**
Fixes #7573 by marking failing tests as unstable.

### CODE REVIEW

File: src/vs/base/parts/storage/test/node/storage.test.ts [hunk 1]
```diff
@@ -206,7 +206,8 @@ suite('Storage Library', () => {
 		await rimraf(storageDir, RimRafMode.MOVE);
 	});
 
-	test('conflicting updates', async () => {
+	// test is disabled due to failures
+	test('conflicting updates @UNSTABLE@', async () => {
```
Comment (aaomidi) [inline src/vs/base/parts/storage/test/node/storage.test.ts]: This tag wont work here. you need to mark it as skip. I think its just test.skip(

@Charles-Gagnon has done that before.

Comment (Charles-Gagnon) [reply src/vs/base/parts/storage/test/node/storage.test.ts]: Yup, just change it to `test.skip('conflicting updates...`

File: src/vs/base/parts/storage/test/node/storage.test.ts [hunk 2]
```diff
@@ -206,7 +206,8 @@ suite('Storage Library', () => {
 		await rimraf(storageDir, RimRafMode.MOVE);
 	});
 
-	test('conflicting updates', async () => {
+	// test is disabled due to failures
```
Comment (Charles-Gagnon) [inline src/vs/base/parts/storage/test/node/storage.test.ts]: Put this on the same line as the edit and add {{ SQL CARBON EDIT}}

`test.skip('conflicting updates', async () => { // {{SQL CARBON EDIT}} test is disabled due to failures`

Comment (Charles-Gagnon) [reply src/vs/base/parts/storage/test/node/storage.test.ts]: (do this for all of them)

Comment (ellbosch) [reply src/vs/base/parts/storage/test/node/storage.test.ts]: Thanks, what does this do?

**Pull request merged**